### ISSUE TITLE
ArgumentListWrapping rule does not detect style violations when initializing a class field

### ISSUE THREAD

**Issue by kruart** [detekt/detekt#7487]
## Expected Behavior
The rule should detect style violations when initializing class fields.

## Observed Behavior
It works when creating a class and assign it to a local variable, but not with global ones, from external scope.

## Steps to Reproduce

Here's the example to reproduce the issue:
```
data class A(val a: String, val b: String, val c: String)

class MyClass {
    companion object {
        const val a = "a"
        const val b = "b"
        const val c = "c"
    }

    var a1: A = A(a, b, c)

    init {
        // THIS ONE IS DETECTED
        val a2 = A(
            a, b,
            c
        )

        // BUT NOT THIS ONE
        a1 = A(
            a, b,
            c
        )
    }
}
```

## Context

Would like to have a consistent code style regardless of whether you're initializing a global or local variable.

## Your Environment

* detekt version 1.23.6
* gradle version 8.4

**Comment by cortinico** [comment]
That's a KtLint wrapping rule, so this issue should be opened against https://github.com/pinterest/ktlint/

**Comment by kruart** [comment]
@cortinico I rechecked the issue with the `ktlint` tool. The problem does not occur with the latest `ktlint`  1.3.1, but it does with version 0.50.0, which the latest `detekt` version is based on.

Seems that `detekt`  already has been moved to the latest `ktlint`  version, so we just need to wait for a new release. This issue should be resolved then. Therefore, I'm closing the ticket.

Thanks for help.
